pages/linux/ceph.md

```diff
@@ -0,0 +1,32 @@
+# ceph
+
+> Ceph implements object storage on a single distributed computer cluster, and provides interfaces for object, block and file-level storage.
+> More information: <https://ceph.io>.
+
+- Check or watch cluster health:
+
+`ceph {{status | -w}}`
+
+- Check cluster usage stats:
+
+`ceph df`
+
+- Get the statistics for the placement groups in your cluster:
+
+`ceph pg dump [--format {{plain | json}}]`
```

Comment:
The syntax make this look quite complicated. What about sticking with something like this?
```suggestion
`ceph pg dump --format {{plain}}`
```

This way, we can indicate that `plain` can be changed, but since JSON output is probably an advanced use-case, we can avoid complicating the page by mentioning it directly.